Calculator/div.py

```diff
@@ -0,0 +1,6 @@
+from numbers import Real
+
+def div(num1, num2):
+    # Check if both inputs are numbers
+    if isinstance(num1, Real) and isinstance(num2, Real):
+        return num1 / num2
```

Comment:
Can you resolve this issue?, is not possible div/0
Traceback (most recent call last):
  File "/Users/cardonita/Documents/Development/learning_python_with_me/Calculator/calculator.py", line 18, in <module>
    print("How much is 3 * 1 = " + div(1,0))
                                   ^^^^^^^^
  File "/Users/cardonita/Documents/Development/learning_python_with_me/Calculator/div.py", line 6, in div
    return num1 / num2
           ~~~~~^~~~~~
ZeroDivisionError: division by zero